src/org/labkey/filetransfer/model/TransferEndpoint.java

```diff
@@ -0,0 +1,66 @@
+package org.labkey.filetransfer.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+
+/**
+ * Created by susanh on 5/14/17.
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TransferEndpoint
+{
+    private String id;
+    private String displayName;
+    private String path;
+    private String localDirectory;
+
+    public TransferEndpoint()
+    {}
+
+    public TransferEndpoint(String id, String path)
+    {
+        setId(id);
+        setPath(path);
+    }
+
+    public String getId()
+    {
+        return id;
+    }
+
+    public void setId(String id)
+    {
+        this.id = id;
+    }
+
+    public String getDisplayName()
+    {
+        return displayName == null ? id : displayName;
+    }
+
+    public void setDisplayName(String displayName)
+    {
+        this.displayName = displayName;
+    }
+
+    public void setDisplay_name(String displayName) { setDisplayName(displayName); }
```

Comment:
The query to the Globus API returns values with underscores between words (Python-style), do this setter is used when deserializing from that call.  We display the endpoint name on the fileTransfer page (if we have it).